purchases/src/customEntitlementComputation/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -227,6 +228,17 @@ class Purchases internal constructor(
         purchasesOrchestrator.invalidateCustomerInfoCache()
     }
 
+    /**
+     * Updates the current appUserID to a new one, without associating the two.
+     *
+     * Important: This method is only available in Custom Entitlements Computation mode.
+     * Tokens posted by the SDK to the RevenueCat backend after calling this method will be sent
+     * with the newAppUserID.
+     */
+    fun switchUser(newAppUserID: String) {
```

Comment:
Just noticed, but we need to add this to the API tests for the customEntitlementComputation flavor.